.github/workflows/tests.yml

```
@@ -0,0 +1,48 @@
+on:
+  push:
+    branches: [ main, master ]
+  pull_request:
+    branches: [ main, master ]
+
+jobs:
+  nslookup-check:
+    runs-on: ubuntu-latest
+    continue-on-error: true  # Equivalent to allow_failures in Travis
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)